modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/display/context/BaseDLEditFileEntryDisplayContext.java

```diff
@@ -73,6 +73,15 @@ public BaseDLEditFileEntryDisplayContext(
 		}
 	}
 
+	@Override
+	public DDMFormValues getDDMFormValues(
+			DDMStructure ddmStructure, Long fileVersionId)
+		throws PortalException {
+
+		return parentDisplayContext.getDDMFormValues(
+			ddmStructure, fileVersionId);
+	}
+
```

Comment:
Why is `fileVersionId` a `Long`? Unless there's a very good reason, you should use primitive types (i.e. `long`).